src/hooks/index.js

```diff
@@ -1,9 +1,13 @@
 import httpHook from './http';
 import { isSwitchedOff, isAwsEnvironment, isHttpWrapped, setHttpWrapped } from '../utils';
+import { hookMongoDb } from './mongodb';
 
 export default () => {
-  if (!isHttpWrapped() && !isSwitchedOff() && isAwsEnvironment()) {
-    httpHook();
-    setHttpWrapped();
+  if (!isSwitchedOff() && isAwsEnvironment()) {
+    if (!isHttpWrapped()) {
+      httpHook();
+      setHttpWrapped();
+    }
+    hookMongoDb();
```

Comment:
1. Doesn't it conflict with the http wrapping?
2. Don't we need to make sure that we don't wrap twice?

Reply:
I will add protection for this.